packages/taquito/src/estimate/rpc-estimate-provider.ts

```diff
@@ -167,6 +169,9 @@ export class RPCEstimateProvider extends OperationEmitter implements EstimationP
       totalStorage += 'originated_rollup' in result ? tx_rollup_origination_size : 0;
     });
 
+    totalStorage +=
+      content.kind === OpKind.SMART_ROLLUP_ORIGINATE ? Number(content.storage_limit) : 0;
```

Comment:
kind of a nitpick, but since we're in the scope of forEach for `result`, we should leverage the values inside that iterated value

Reply:
oh thanks for hte comment i started working on this before which should take in a constant but was put on hold until  the other pr was merged

Reply:
this has changed but comment still applies 
the check is for content which operationResults does not contain and unsure if there are every conditions where tehre are multiple values inside of the operation restults that could cause for the storage limit to be added multiple times